app/controllers/home_controller.rb

```diff
@@ -2,13 +2,29 @@ class HomeController < ApplicationController
   before_action :authenticate_client!
 
   def index
+    @products = Product.page(params[:page]).per(10)
+  end
+
+  def filter
     @products = Product.all
+
+    filter_params.each do |key, value|
+      @products = @products.where(key => true) if value == '1'
+    end
+
+    @products = @products.page(params[:page]).per(10)
+    render :index
```

Comment:
Let's try to keep to restful routes. To filter the products we should just send the filter params to the index action

Reply:
I've covered this on this commit [9375356](https://github.com/AguirreCampiNicolas/onboarding/pull/1/commits/937535640e112d2f9e0a82c57177df8c0228c24c)